setup.py

```diff
@@ -150,6 +150,26 @@ def __setattr__(self, k, v):
 
 
 def get_setup_args():
+    extension = Extension(
```

Comment:
It looks like these additions don't follow the existing conventions around spaces, so please fix that.